src/Akka.Streams.Kafka/Stages/Consumers/Abstract/SubSourceLogic.cs

```diff
@@ -450,191 +565,228 @@ public SubSourcePromiseControl(SourceShape<(
             /// <inheritdoc />
             public override void PerformShutdown(Exception? ex) => _performShutdown(ex);
         }
+    }
 
-        private class SubSourceStreamStage : GraphStage<SourceShape<TMessage>>
```

Comment:
Moved this class outside of its parent - we had nested classes like 4 layers deep in this file.